src/h5web/visualizations/shared/Tooltip.tsx

```diff
@@ -3,14 +3,19 @@ import { Dom, PointerEvent, useThree } from 'react-three-fiber';
 import { TooltipWithBounds, useTooltip } from '@vx/tooltip';
 import { scaleLinear } from 'd3-scale';
 import { format } from 'd3-format';
-import styles from './HeatmapVis.module.css';
-import { useProps } from './hooks';
+import styles from './Tooltip.module.css';
+import { Domain } from './models';
 
 type Coords = [number, number];
 
-function Tooltip(): ReactElement {
-  const { dims, data } = useProps();
-  const [rows, cols] = dims;
+interface Props {
+  xDomain: Domain;
+  yDomain: Domain;
+  data: number[][];
+}
+
+function Tooltip(props: Props): ReactElement {
+  const { xDomain, yDomain, data } = props;
```

Comment:
To be more generic, `Tooltip` now works with a domain for `x` and for `y` rather than `[rows, cols]`. This allows to have domain that do not start at 0.

Reply:
This was reverted due to https://github.com/silx-kit/h5web/pull/108#pullrequestreview-403634393.